src/hooks/useLoadGlobalLateData.js

```diff
@@ -27,10 +31,11 @@ export default function useLoadGlobalLateData() {
     const promises = [];
     const p1 = loadWalletBalanceNamesToCache();
     const p2 = dispatch(nonceManagerLoadState());
-    promises.push(p1, p2);
+    const p3 = prefetchAccountENSDomains({ accountAddress });
```

Comment:
I don't think this should be here.  We made it global on purpose to separate from account specific actions.
If you don't find anywhere else where it fits, maybe we need a new hook like `useLoadAccountLateData`